plugin/storage/badger/spanstore/read_write_test.go

```diff
@@ -687,8 +688,6 @@ func TestRandomTraceID(t *testing.T) {
 		}
 		traces, err := sr.FindTraces(context.Background(), params)
 		assert.NoError(t, err)
-
-		// failed with `second` tag query, but success with `first`
```

Comment:
Think it would be a good idea to do a separate query for tag 'first' as well - just to ensure that both work?

Reply:
The other tests actually catch that case.

Reply:
ok thanks